src/plugins/data/server/search/collectors/usage.ts

```diff
@@ -71,7 +70,7 @@ export function usageProvider(core: CoreSetup<object, DataPluginStart>): SearchU
   };
 
   return {
-    trackError: getTracker('errorCount'),
+    trackError: () => getTracker('errorCount')(),
```

Comment:
We don't really have an accurate way to see the time a failure took since there is no `took` in the response for a failure. So this was just to make typescript happy since the error handler for `errorCount` didn't accept a `duration`.